examples/low_level_api.py

```diff
@@ -3,46 +3,59 @@
 import logging
 import argparse
 from dotenv import load_dotenv
+from openai import OpenAI
+
 from personal_graph import database as db
 from personal_graph import visualizers
+from personal_graph.embeddings import OpenAIEmbeddingsModel
 
 
-def insert_single_node(db_url, auth_token, new_label, new_node, new_node_id):
+def insert_single_node(
+    db_url, auth_token, embedding_model, new_label, new_node, new_node_id
```

Comment:
Might as well create a db client and pass it around, this way managing local and remote db simulataneously would be easier